pbj-core/gradle.properties

```diff
@@ -1,5 +1,5 @@
 # Version number
-version=0.1.0-SNAPSHOT
+version=0.3.3.11
```

Comment:
Not a valid semantic version and will fail on Maven Central and Gradle Plugins release.